src/Manager/Environment/Install.tsx

```diff
@@ -14,34 +19,80 @@ import Button from './Button';
 import { install } from './effects/installEnvironment';
 import { isOnlyAvailable, version } from './environmentReducer';
 
-type Props = { environment: Environment };
+type Props = { environment: Environment; showExperimentalWarning: boolean };
 
-const Install = ({ environment }: Props) => {
+const Install = ({ environment, showExperimentalWarning }: Props) => {
     const dispatch = useDispatch();
     const anyInProgress = useSelector(isAnyToolchainInProgress);
+    const [showExperimentalDialog, setShowExperimentalDialog] = useState(false);
 
     const { platform } = process;
-    const onClick = (() => {
+    const onClick = () => {
         switch (platform) {
             case 'darwin':
-                return () => dispatch(install(environment, false));
+                return dispatch(install(environment, false));
             case 'linux':
-                return () =>
-                    dispatch(showConfirmInstallDirDialog(version(environment)));
+                return dispatch(
+                    showConfirmInstallDirDialog(version(environment))
+                );
             case 'win32':
-                return () =>
-                    dispatch(showConfirmInstallDirDialog(version(environment)));
+                return dispatch(
+                    showConfirmInstallDirDialog(version(environment))
+                );
         }
-    })();
+    };
 
     return !isOnlyAvailable(environment) ? null : (
-        <Button
-            icon="x-mdi-briefcase-download-outline"
-            onClick={onClick}
-            label="Install"
-            variant="secondary"
-            disabled={anyInProgress}
-        />
+        <>
+            <GenericDialog
+                isVisible={showExperimentalDialog}
+                headerIcon="alert"
+                title="Support for Toolchain and SDK experimental in VS Code"
+                footer={
+                    <>
+                        <DialogButton
+                            variant="danger"
+                            onClick={() => {
+                                setShowExperimentalDialog(false);
+                                onClick();
+                            }}
+                        >
+                            Accept the risks and install
+                        </DialogButton>
+                        <DialogButton
+                            onClick={() => {
+                                setShowExperimentalDialog(false);
+                                openUrl(
+                                    'https://github.com/microsoft/vscode-docs/blob/vnext/release-notes/v1_63.md#pre-release-extensions'
```

Comment:
Assuming we need to provide the link about how to switch to pre-release, here's the link we're going to use:
```suggestion
                                    'https://docs.nordicsemi.com/bundle/nrf-connect-vscode/page/rel_notes_overview.html'
```

Reply:
Push missing?